index.js

```diff
@@ -0,0 +1,21 @@
+$(document).ready(function () {
+  var searchTerm = $('#searchTerm');
+  $('#submitBtn').click(function (event) {
+    event.preventDefault();
+    $("#output").empty();
+    var username = searchTerm.val();
+    var url = 'https://en.wikipedia.org/w/api.php?origin=*&action=query&format=json&list=usercontribs&ucuser={{searchTerm}}&ucdir=newer&uclimit=5&callback=?'.replace('{{searchTerm}}', username);
```

Comment:
If you want to use template-like notation (which is nice) but don't want to pull in a templating library just because of that (which is reasonable), you could write a simple helper function to do the replacement.

Whether you do that or not, make sure to add proper escaping. Usernames can't contain HTML but they can contain special characters like `&` or `?` (and it's good practice to assume they could contain anything, anyway).

Reply:
Also, there isn't much point in using both `origin=*` (which is for CORS) and `callback=?` (which forces the request to be JSONP).

Reply:
can you please give some examples on proper escaping about what data has to be prevented?

callback=? has been removed , further origin=* is to test the project locally.

Reply:
Let's say the user is called `A&B`, then the URL would be `+    var url = 'https://en.wikipedia.org/w/api.php?origin=*&action=query&format=json&list=usercontribs&ucuser=A&B&ucdir=newer&uclimit=5`. A webserver will interpret that like 
<pre>
origin  => *
action  => query
format  => json
list    => usercontribs
ucuser  => A
B       =>
ucdir   => newer
uclimit => 5
 </pre>
which refers to a different user.

Reply:
@tgr sir,
I have updated my js file and resolved this issue using encodeURI function .Please review it.
Due to some technical issue I could not upload my file on time , sorry for the delay.